plugins/redfish/fu-redfish-device.c

```diff
@@ -504,6 +526,10 @@ fu_redfish_device_probe(FuDevice *dev, GError **error)
 		}
 	}
 
+	/* for Supermicro check whether we have a proper Redfish license installed */
+	if (g_strcmp0("SMCI", fu_device_get_vendor(dev)) == 0)
```

Comment:
do you ever OEM systems that this wouldn't be valid?  If not, then this is probably fine.  If you do, I would wonder if it's better to set some metadata or check the device gtype.